src/nvim/po/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ if(HAVE_WORKING_LIBINTL AND GETTEXT_FOUND AND XGETTEXT_PRG AND ICONV_PRG)
         -DXGETTEXT_PRG=${XGETTEXT_PRG}
         -DPOT_FILE=${NVIM_POT}
         -DSEARCH_DIR=${CMAKE_CURRENT_SOURCE_DIR}
-        "'-DSOURCES=${NEOVIM_RELATIVE_SOURCES}'"
+        "\"-DSOURCES=${NEOVIM_RELATIVE_SOURCES}\""
```

Comment:
@jszakmeister I tried that but `build/src/nvim/po/nvim.pot` didn't generate correctly. My assumption was that you were intending to send the literal string

    "-DSOURCES=${NEOVIM_RELATIVE_SOURCES}"

for some reason?